test/js/inline-style-property-map.js

```diff
@@ -21,6 +21,16 @@ suite('Inline StylePropertyMap', function() {
     assert.strictEqual(this.element.style['height'], simpleLength.cssString);
   });
 
+  test('The set method successfully sets the CSS string of the list of StyleValues on property ' +
```

Comment:
This seems unrelated? If you have changes that you would like to make to unrelated parts of the codebase, it's good practice to put them in separate pull requests.